updatecli/updatecli.d/updatek3sroot.yaml

```diff
@@ -0,0 +1,66 @@
+---
+name: "Update k3sroot version" 
+
+sources:
+ k3sroot:
+   name: Get k3s-root version
+   kind: githubrelease
+   spec:
+     owner: k3s-io
+     repository: k3s-root
+     token: '{{ requiredEnv "GITHUB_TOKEN" }}'
```

Comment:
I think this is not going to work the env defined on the updatecli action is GH_TOKEN

Reply:
you are right, I'm dumb. Fixed!

Reply:
Good catch

According 
https://github.com/rancher/image-build-flannel/blob/b5f8809a25d349a6ef3ca12d9c4d9ba1584b2285/.github/workflows/updatecli.yml#L55

It should be `UPDATECLI_GITHUB_TOKEN`

Reply:
```suggestion
     token: '{{ requiredEnv "UPDATECLI_GITHUB_TOKEN" }}'
```